### ISSUE TITLE
Training time？

### ISSUE THREAD

**Issue by zihaozhang9** [liuzechun/MetaPruning#25]
How long does it take to train PrunedNet?How many GPU hours?

**Comment by ccsone** [comment]
张兄，你最近也研究这个？ 一起交流交流

**Comment by zihaozhang9** [comment]
[8 1080Ti GPUs.](https://github.com/liuzechun/MetaPruning/issues/2#issuecomment-519879543) [3 days](https://github.com/liuzechun/MetaPruning/issues/8#issuecomment-529083248).too long.

**Comment by zihaozhang9** [comment]
> 张兄，你最近也研究这个？ 一起交流交流

无没有条件尝试这个代码库。他们使用了太多gpus，我没有

**Comment by ccsone** [comment]
@zihaozhang9 可以不训练这么多的类别，这样也可以吧

**Comment by zihaozhang9** [comment]
> @zihaozhang9 可以不训练这么多的类别，这样也可以吧

你可以尝试一下。
有时候训练时间长，除了可能数据集大，还有可能是网络模型大，所以必须更大的显存才能载入整个网络。还要考虑，修改了batch_size，是否需要修改学习率。这些变化可能会导致训练结果不一样。
如果减少了类别，剪枝的网络也会发生变化。

**Comment by ccsone** [comment]
@zihaozhang9 嗯  我跑search  的时候    memory  就不够了

**Comment by ccsone** [comment]
@zihaozhang9  你注意到  裁剪的ratio  在哪里吗

**Comment by liuzechun** [comment]
Hi, for the correlation between reducing the batchsize and the learning rate, maybe you can also refer to issue #2 
I am sorry we do use 8 1080Ti and the whole pipeline takes 3 days. I think it is already a very fast algorithm, but I also understand it is hard to carry out NAS in academia with limited GPUs. Hope there is any thing I can help. If you have any further questions regarding how to reduce the batch size, please feel free to ask! I will try to reply you as soon as possible!
